src/shared/lib/Message/NoResultsMessage.js

```diff
@@ -5,14 +5,22 @@ import type { Props } from '../../../flow/types.js.flow';
 import css from '../Style/NoResultsMessage.css';
 
 const NoResultsMessage = ({ ...props }: Props) => {
-  const { translate } = props;
+  const { translate, store } = props;
+  const filterText = store.getState().form.searchForm.values.searchTextArea;
+
+  let query = translate({ id: 'ui-marccat.error.noresults.message' }) + '. ';
+  if (filterText.length > 0) {
+    query = translate({ id: 'ui-marccat.error.noresultsfoundfor.message' }) + ' "' + filterText + '". ';
```

Comment:
[String Interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) would make the code easier to read here and on line 11.

Reply:
ok Ryan, i have changed the code